src/layout/MultipleSelect/index.tsx

```diff
@@ -51,7 +53,42 @@ export class MultipleSelect extends MultipleSelectDef {
     return useEmptyFieldValidationOnlySimpleBinding(node);
   }
 
+  renderLayoutValidators(props: NodeValidationProps<'MultipleSelect'>): JSX.Element | null {
+    return <MultipleSelectLayoutValidator {...props} />;
+  }
+
   validateDataModelBindings(ctx: LayoutValidationCtx<'MultipleSelect'>): string[] {
-    return this.validateDataModelBindingsSimple(ctx);
+    const errors: string[] = [];
```

Comment:
Instead of copy-pasting this, you can also extract it to a function in a separate file and import it. :pray:

Reply:
I took care of this now, resolving this.